CONTRIBUTING.md

```diff
@@ -261,9 +267,9 @@ Had no year associated with it, so it has keywords for its tag `pca-visually-exp
 
 #### Spell checking
 
-Spell checks are run automatically using Github actions upon opening a PR for master or prior to merging to master.
-Github actions will abort if there are more than 2 spelling errors and you will need to fix those before continuing.
-You can obtain the list of spelling errors on Github by going to `Actions` and clicking the workflow of PR you are working on.
+Spell checks are run automatically using GitHub actions upon opening a PR for master or prior to merging to master.
+GitHub actions will abort if there are more than 2 spelling errors and you will need to fix those before continuing.
+You can obtain the list of spelling errors on GitHub by going to `Actions` and clicking the workflow of PR you are working on.
 Click on the `style-n-check` step and in the upper right hand corner, there is a button that says "Artifacts" which should list a file called `spell-check-results`.
 Click on `spell-check-results` to download a zip file that contains the list of misspelled words.
 Alternatively, click on the "Check on spell check results" step in the workflow log to see the misspellings.
```

Comment:
Re the instructions to run the R script and perform spell check locally at line 280 (it won't allow me to leave my comment there in this PR), I had to install the `spelling` package due to the following error:

`Error in loadNamespace(name) : there is no package called ‘spelling’
Calls: :: ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted`

while running in the Docker container.

If the intent is to leave this package out of the Docker container, then we should probably add a note here about likely needing to install the package if you go the local route first. (I could have also done something wrong but I rebuilt the docker image, as well as pulled the latest, and got the error both times).

Reply:
This is odd. I've been running this on my Docker container without issue, but I do see there is no explicit install of `spelling`. Let me pull the most recent docker image and start fresh and see whats up. (I usually make it a point to not install things manually to container but always install using the Dockerfile build).

Reply:
You are correct, it is not on the Docker container. Good catch. I forgot that for github actions we have install specifically. Will add this to Dockerfile now. (Should push automatically I think based on #237 #252)